src/psfmachine/tpf.py

```diff
@@ -283,6 +294,130 @@ def plot_tpf(self, tdx):
                 edgecolor="k",
             )
 
+    def load_shape_model(self, plot=False, input=None):
+        """
+        Loads a PRF shape model from files, based on FFI models.
+        """
+
+        # load model hyperparams from file
+        channel = self.tpf_meta["channel"][0]
+        quarter = self.tpf_meta["quarter"][0]
+        # use FFI models
+        if input is None:
+            input = "%s/data/ffi_prf_models_v0.1.0.csv" % (PACKAGEDIR)
+
+        # check if file exists
+        if not os.path.isfile(input):
+            raise FileNotFoundError("No PSF files: ", input)
+
+        try:
+            tab = pd.read_csv(input, index_col=0, header=[0, 1])
+            self.n_r_knots = int(tab.loc[channel, (str(quarter), "n_r_knots")])
+            self.n_phi_knots = int(tab.loc[channel, (str(quarter), "n_phi_knots")])
+            # *4 to convert from pixel to arcseconds
+            self.rmin = int(tab.loc[channel, (str(quarter), "rmin")] * 4)
```

Comment:
This *4 is only valid when we're using a kepler-apertures format...we need to be careful here.

Reply:
implemented here:
11d13f2b5269e39799ae72b5636abbce67a1783d